charts/prometheus-postgresql-alerts/values.yaml

```
@@ -11,99 +11,90 @@ rulesGroupName: postgresql.rules
 rules:
 
   PostgreSQLExporterDown:
```

I would suggest to remove references to PostgreSQL exporter in the name too (Need to rename the runbook too) 
```suggestion
  SQLExporterDown:
```

---

I had concerns about undertaking such a significant refactoring in case it caused further disruptions, but your suggestion seems appropriate in some cases (i wouldn't update "pg_" related alerts name).
We'd have:
* SQLExporterSomething: for SQLExporter related alerts (ex: target down)
* PostgreSQLExporterSomething: for pg related alerts (ex: long-running query)